src/main/java/amazin/model/Book.java

```diff
@@ -23,7 +23,7 @@
     @NotNull
     @Size(min = 13, max = 13, message = "ISBN should be 13 characters long") // ISBN is a 13 digit number
     private String ISBN;
-    private String picture;
```

Comment:
weird that we had it as a picture before but storing it as a String, this makes much more sense now as a url!